api/upgradeseries/upgradeseries_test.go

```diff
@@ -0,0 +1,4 @@
+// Copyright 2018 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package upgradeseries_test
```

Comment:
Required?

Reply:
I will add some tests for this.

Reply:
I'm assuming the package was created before tests were added and @SimonRichardson was simply asking if it is required to make an empty package with no tests. The tests were subsequently added.